packages/ide-extension/src/panel/guidedAnswersPanel.ts

```diff
@@ -54,10 +56,11 @@ export class GuidedAnswersPanel {
          *
          * const webappDirPath = dirname(require.resolve('@sap/guided-answers-extension-webapp'));
          */
+        const ViewColumnType = openToSide ? ViewColumn.Beside : ViewColumn.Active;
```

Comment:
Hi @sanat-thukral, could we have a unit test covering this?

Reply:
Sure thing Klaus